arrows/vxl/CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ kwiver_add_library( kwiver_algo_vxl
 target_link_libraries( kwiver_algo_vxl
   PUBLIC               vital_algo
                        vil vpgl vgl
-  PRIVATE              vital_klv
+  PRIVATE              kwiver_algo_klv
```

Comment:
This is a build error (VXL enabled, KLV not enabled) waiting to happen...

Reply:
Here's the most relevant example I know, which happens to be what I mentioned in the working group: [throw a fatal erro](https://github.com/Kitware/kwiver/blob/536a0421e2f5f6634aeb36571b6f3766d3d23a04/arrows/vtk/applets/CMakeLists.txt#L19)